File: Unidad 3/U3programa22.cs
using System;

namespace U3programa22 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int suma = 0, num;
            do
            {
                Console.WriteLine("Ingrese el valor de num: ");
                num = Convert.ToInt32(Console.ReadLine());
                suma = suma + num;
            } while (num != 0);
            Console.WriteLine("El resultado es: " + suma);
        }
    }
}

File: Unidad 3/U3programa22.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3programa22;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class U3programa22 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int suma=0, num;
        do
        {
            System.out.println("Ingrese el valor de num: ");
            num = sc.nextInt();
            suma = suma + num;
        }while (num != 0);
        System.out.println("El resultado es: " + suma);
    }
}
